wine.def:1 syntax error on 32-bit build script

Hi, I've successfully run the 64-bit build script on Debian 9 x64. I am now trying to do exactly the same on my Debian 9 i386 with the 32-bit script, but I get the following linker error coming to the end:

> /usr/bin/ld:../../../wine-git/libs/wine/wine.def: file format not recognized; treating as linker script
> /usr/bin/ld:../../../wine-git/libs/wine/wine.def:1: syntax error
> collect2: error: ld returned 1 exit status
> Makefile:375: recipe for target 'libwine.dll' failed
> make[1]: *** [libwine.dll] Error 1
> make[1]: Leaving directory '/home/francisco/Downloads/wine-win32/libs/wine'
> Makefile:19509: recipe for target 'libs/wine' failed
> make: *** [libs/wine] Error 2
> Wine make failed with error 0

Any ideas? Thanks in advance.

--- a/32 bit/buildd3d_all.sh
+++ b/32 bit/buildd3d_all.sh
@@ -31,7 +31,7 @@ then
 fi
 rm -f test_wined3d
 echo Downloading system updates and required dependencies...
-apt-get update -qq -y && apt-get dist-upgrade -qq -y && apt-get build-dep wine -qq -y && apt-get install mingw32 git -qq -y && apt-get autoremove -qq -y && apt-get clean -qq -y
+apt-get update -qq -y && apt-get dist-upgrade -qq -y && apt-get build-dep wine -qq -y && apt-get install mingw-w64 git -qq -y && apt-get autoremove -qq -y && apt-get clean -qq -y
 if [ $? -ne 0 ]
 then
 	echo Download failed with error $?
@@ -75,7 +75,7 @@ then
 	exit 4
 fi
 cd ../wine-win32
-../$p/configure --without-x --without-freetype --host=i586-mingw32msvc CFLAGS="-O2 -DWINE_NOWINSOCK -DUSE_WIN32_OPENGL" --with-wine-tools=../wine-tools/ LDFLAGS=" -static-libgcc"
+../$p/configure --without-x --without-freetype --host=i686-w64-mingw32 CFLAGS="-O2 -DWINE_NOWINSOCK -DUSE_WIN32_OPENGL" --with-wine-tools=../wine-tools/ LDFLAGS=" -static-libgcc"
 if [ $? -ne 0 ]
 then
 	echo Wine configure failed with error $?
@@ -124,7 +124,7 @@ then
 	exit 9
 fi
 cd ../wine-win32
-../$p2/configure --without-x --without-freetype --host=i586-mingw32msvc CFLAGS="-O2 -DWINE_NOWINSOCK -DUSE_WIN32_OPENGL" --with-wine-tools=../wine-tools/ LDFLAGS=" -static-libgcc"
+../$p2/configure --without-x --without-freetype --host=i686-w64-mingw32 CFLAGS="-O2 -DWINE_NOWINSOCK -DUSE_WIN32_OPENGL" --with-wine-tools=../wine-tools/ LDFLAGS=" -static-libgcc"
 if [ $? -ne 0 ]
 then
 	echo Wine configure failed with error $?

